features/order/managing_orders/modifying_shipping_address/modifying_shipping_address_on_order_with_promotion.feature

```diff
@@ -0,0 +1,31 @@
+@managing_orders
+Feature: Modifying a customer's shipping address on an order with an applied promotion
+    In order to ship an order to a correct place
+    As an Administrator
+    I want to be able to modify a customer's shipping address without changing an order's total
+
+    Background:
+        Given the store operates on a single channel in "United States" named "Web"
+        And the store ships everywhere for free
+        And the store allows paying with "Cash on Delivery"
+        And the store has a product "Suit" priced at "$400.00"
+        And it belongs to "Suits"
+        And there is a promotion "Holiday promotion"
+        And the promotion gives "$50.00" off if order contains products classified as "Suits"
+        And there is a customer "mike@ross.com" that placed an order "#00000001"
+        And the customer bought a single "Suit"
+        And the customer "Mike Ross" addressed it to "350 5th Ave", "10118" "New York" in the "United States" with identical billing address
+        And the customer chose "Free" shipping method with "Cash on Delivery" payment
+        And I am logged in as an administrator
+
+    @todo
+    Scenario: Modifying a customer's shipping address when the applied promotion is no longer valid
+        Given the promotion was disabled for the channel "Web"
```

Comment:
For 1.0 we deliver simple shipping address edit feature which will only modify the shipping address and leave the rest of the order untouched. That means the order will not be recalculated and this scenario covers situation when the promotion that was applied to an order is disabled after order was placed. It ensures the promotion is still applied to an order after shipping address modification.